gravitl/netclient
# fix(GRA-919): use netmaker logo

TESTING STEPS
1. build netclient gui
2. run the gui
3. check that the netmaker logo shows as the app icon (may not work on GNOME as an extra .desktop file may be needed)
